angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/option-link.html

```diff
@@ -17,6 +16,5 @@
            <option ng-repeat='obj in portlet.widgetData[config.arrayName]' value="{{obj[config.value]}}" title='{{obj[config.display].length > 26 ? obj[config.display] : ""}}'>{{obj[config.display] | trimMiddle:26}}</option>
        </select>
     </div>
-  </div>
```

Comment:
I experimented with where to put the href until I found the right spot - but I left the remains of my experiments scattered through the code while I was committing. Consequently, I committed some div tags I shouldn't have.